bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -1,10 +1,16 @@
 #GoAndKickBallMapGoal
-$AlignedToGoal
-    YES --> $BallKickArea
-        LEFT --> @LookAtBall, @StandAndWait + duration:1, @KickBallDynamic
-        RIGHT --> @LookAtBall, @StandAndWait + duration:1, @KickBallDynamic
-        FAR -->@LookAtFieldFeatures, @GoToBall + target:map_goal
-    NO -->  @GoToBall + target:map_goal
+$AvoidBall
+    NO --> $BallClose + distance:1
+        YES --> $AlignedToGoal
+            YES --> $BallKickArea
+                LEFT --> @Stop, @LookAtBall, @StandAndWait + duration:1, @KickBallDynamic
+                RIGHT --> @Stop, @LookAtBall, @StandAndWait + duration:1, @KickBallDynamic
```

Comment:
with the dynamic kick, I think the behavior does not have to decide for a foot.

Reply:
That's right, but we would need to modify the decision for this and have a look if its used somewhere where `RIGHT` and `LEFT` are used.

Reply:
We move this to a later pr